fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -353,14 +365,15 @@ void FixedLagSmoother::processQueue(fuse_core::Transaction& transaction)
     {
       // The motion model processing failed.
       // Check the transaction timeout to determine if it should be removed or skipped.
-      if (element.transaction->stamp() + params_.transaction_timeout < current_time)
+      const auto& max_stamp = element.maxStamp();
+      if (max_stamp + params_.transaction_timeout < current_time)
       {
         // Warn that this transaction has expired, then skip it.
-        ROS_ERROR_STREAM("The queued transaction with timestamp " << element.transaction->stamp() <<
-                          " from sensor " << element.sensor_name << " could not be processed after " <<
-                          (current_time - element.transaction->stamp()) << " seconds, which is greater " <<
-                          "than the 'transaction_timeout' value of " << params_.transaction_timeout <<
-                          ". Ignoring this transaction.");
+        ROS_ERROR_STREAM("The queued transaction with timestamp " << element.stamp() << "and maximum "
```

Comment:
A white space is missed before `and`:
```suggestion
        ROS_ERROR_STREAM("The queued transaction with timestamp " << element.stamp() << " and maximum "
```